example/lib/src/lists.dart

```diff
@@ -298,8 +298,7 @@ class _ImageListDemoState extends State<ImageListDemo> {
   void initState() {
     super.initState();
     _selectionController = ListViewSelectionController();
-    _disablerController = ListViewItemDisablerController()
-      ..filter = (int index) => index == 2 || index == 3;
+    _disablerController = ListViewItemDisablerController()..filter = (int index) => index == 2 || index == 3;
```

Comment:
I'm curious what line length you format this repo at. 80 was too short, but 120 seems too long. Happy to reformat to what ever length is appropriate.